spacy/tests/test_cli.py

```diff
@@ -1042,67 +864,6 @@ def test_applycli_user_data():
         assert result[0]._.ext == val
 
 
-# TODO: remove this xfail after merging master into v4. The issue
-#       is that for local files, pathy started returning os.stat_result,
-#       which doesn't have a last_modified property. So, recency-sorting
-#       fails and the test fails. However, once we merge master into
-#       v4, we'll use weasel, which in turn uses cloudpathlib, which
-#       should resolve this issue.
-@pytest.mark.xfail(reason="Recency sorting is broken on some platforms")
```

Comment:
Looks like this entire tests got deleted?

Reply:
Yeah, this test is weasel now.